F/Fmt/build_tarballs.jl

```diff
@@ -0,0 +1,58 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "Fmt"
+version = v"9.1.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/fmtlib/fmt.git", "a33701196adfad74917046096bf5a2aa0ab0bb50")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+INSTALL_PATH=${prefix}
+cd $WORKSPACE/srcdir/fmt
+mkdir build
+cd build
+cmake \
+-DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} \
+-DBUILD_SHARED_LIBS=True \
+-DCMAKE_INSTALL_PREFIX=${INSTALL_PATH} \
+..
```

Comment:
Why defining a variable, used only once, equal to another variable?  Why not directly using that variable, like in all other recipes here?  Also, you keep not following the [documentation](https://docs.binarybuilder.org/stable/build_tips/#CMake-builds), use also `-DCMAKE_BUILD_TYPE=Release`: 
```suggestion
cd $WORKSPACE/srcdir/fmt
mkdir build
cd build
cmake \
    -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} \
    -DCMAKE_BUILD_TYPE=Release \
    -DBUILD_SHARED_LIBS=True \
    -DCMAKE_INSTALL_PREFIX=${prefix} \
    ..
```